src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -68,10 +68,42 @@ By including and configuring Sentry, the SDK will automatically attach global ha
 
 [{% asset js-index/automatically-capture-errors.png alt="Stack trace of a captured error" %}]({% asset js-index/automatically-capture-errors.png @path %})
 
-Keep in mind; browsers are taking some security measures when serving script files from different origins. For errors to always make their way to Sentry, make sure that you configure CORS headers and add appropriate script attributes. 
+Browsers take security measures when serving script files from different origins. To ensure errors are routed to Sentry, configure the CORS headers, and add script attributes. 
```

Comment:
>  To ensure errors are routed to Sentry, configure the CORS headers, and add script attributes.

"The CORS headers" isn't correct; drop "the". "Appropriate script attributes" is more accurate than "add script attributes". The second half of this sentence was more accurate before edits.

Reply:
@benvinegar, I updated the sentence. 😄 Let me know if there are any other changes you'd like.